modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseInit.java

```diff
@@ -0,0 +1,40 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+public class ClickHouseInit {
+    public static final String JDBC_NAME_YANDEX = "clickhouse";
+    public static final String JDBC_NAME_MYSQL = "mysql";
+
+    public static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("yandex/clickhouse-server");
+
+    public static final String CLICKHOUSE_DRIVER_CLASS_NAME = "ru.yandex.clickhouse.ClickHouseDriver";
+    public static final String MYSQL_DRIVER_CLASS_NAME = "com.mysql.cj.jdbc.Driver";
+
+    public static final Integer HTTP_PORT = 8123;
+    public static final Integer NATIVE_PORT = 9000;
+    public static final Integer MYSQL_PORT = 9004;
+
+    public static final String databaseName = "default";
+    public static final String username = "default";
+    public static final String password = "";
```

Comment:
Why aren't those uppercase?

Reply:
Because I have been doing Go for too long ... geez, how stuff like that seeps into daily coding.
Will change.

Reply:
Fixed.